### DISCUSSION TITLE
Some questiones about the description "When a Constellation node image boots inside a CVM, ..."

### DISCUSSION THREAD

**Discussion by fffmonkeyking** [edgelesssys/constellation#3452] [Q&A] [answered]
### Issue description

Some questiones about the description "When a Constellation node image boots inside a CVM, ..." on https://docs.edgeless.systems/constellation/next/architecture/attestation#node-attestation:

1、How to boot a Constellation node image in a CVM for Constellation ?

2、Is the launched Constellation node image treated as a Constellation node or is the CVM treated as a Constellation  node?

3、Is Constellation running multiple k8s pods in the Constellation node？

**Comment by 3u13r** [comment, marked as answer]
Hello,

> 1、How to boot a Constellation node image in a CVM for Constellation ?

Constellation is primarily build to be deployed in AWS, Azure, or GCP. We leverage their CVM offerings and start up VMs with confidential computing enabled. If you follow our quick-start guide you will deploy a Constellation including CVMs.
If you just want to see the general architecture of your VM images, then you can have a look at the "local deployment" called miniconstellation. This will use KVM to start up 2 VMs/nodes on your local machine but with CC disabled.

> 2、Is the launched Constellation node image treated as a Constellation node or is the CVM treated as a Constellation node?

The Constellation node image is a VM image. Therefore, a Constellation node and a CVM have a 1:1 relationship and the terms can be used interchangeably.

 > 3、Is Constellation running multiple k8s pods in the Constellation node？

Yes, as Constellation nodes are Kubernetes nodes, you can run multiple pods inside each node.
